core/pom.xml

```diff
@@ -1,38 +1,103 @@
+  </parent>
+  <artifactId>time-tracker-core</artifactId>
+  <packaging>jar</packaging>
+  <name>Time Tracker (Core)</name>
+  <dependencies>
+    <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-context</artifactId>
+      <version>5.3.10</version>
+    </dependency>
+    <!-- Testing -->
+    <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-test</artifactId>
+      <version>${spring.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.12</version>
+      <scope>test</scope>
+    </dependency>
+    <!-- End Testing -->
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-api</artifactId>
+      <version>5.8.1</version>
+      <scope>test</scope>
+    </dependency>
+    <!-- Dependency added by Roost-->
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-params</artifactId>
+      <version>5.8.1</version>
+      <scope>test</scope>
+    </dependency>
+    <!-- Dependency added by Roost-->
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-engine</artifactId>
+      <version>5.8.1</version>
+      <scope>test</scope>
+    </dependency>
+    <!-- Dependency added by Roost-->
+    <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-core</artifactId>
+      <version>5.3.10</version>
+      <scope>compile</scope>
+    </dependency>
+    <!-- Dependency added by Roost-->
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-core</artifactId>
+      <version>3.12.4</version>
+      <scope>test</scope>
+    </dependency>
+    <!-- Dependency added by Roost-->
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-all</artifactId>
+      <version>1.10.19</version>
+      <scope>test</scope>
+    </dependency>
+    <!-- Dependency added by Roost-->
+    <dependency>
+      <groupId>training.taylor.timetracker</groupId>
+      <artifactId>timetracker-core</artifactId>
+      <version>0.6.0-SNAPSHOT</version>
+      <scope>compile</scope>
+    </dependency>
+    <!-- Dependency added by Roost-->
+    <dependency>
+      <groupId>java.util</groupId>
+      <artifactId>ArrayList</artifactId>
+      <version>1.8</version>
+      <scope>compile</scope>
+    </dependency>
+    <!-- Dependency added by Roost-->
+    <dependency>
+      <groupId>java.util</groupId>
+      <artifactId>List</artifactId>
+      <version>1.8</version>
+      <scope>compile</scope>
+    </dependency>
+    <!-- Dependency added by Roost-->
+    <dependency>
+      <groupId>org.junit</groupId>
+      <artifactId>Assert</artifactId>
+      <version>4.12</version>
+      <scope>test</scope>
+    </dependency>
+    <!-- Dependency added by Roost-->
+  </dependencies>
```

Comment:
The dependencies added by roost are all wrong, All of them are giving missing artifact error